File: android-workspace/Hello/app/src/main/java/com/ririnto/app/hello/MainActivity.java
package com.ririnto.app.hello;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        TextView mTv = findViewById(R.id.mTv);
        Button mBtn = findViewById(R.id.mBtn);
        mBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                if (mTv.getText() == "Good Evening") {
                    mTv.setText("Good Morining");
                } else {
                    mTv.setText("Good Evening");
                }
            }
        });
    }
}

File: 2021/java/android-workspace/hellokt/app/src/main/java/com/ririnto/app/hellokt/MainActivity.kt
package com.ririnto.app.hellokt

import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val mTv = findViewById<TextView>(R.id.mTv)
        val mBtn = findViewById<Button>(R.id.mBtn)
        mBtn.setOnClickListener {
            if (mTv.text === "Good Evening") {
                mTv.text = "Good Morining"
            } else {
                mTv.text = "Good Evening"
            }
        }
    }
}